Rakefile

```diff
@@ -103,3 +103,8 @@ task(:commits) do
     puts "...passed"
   end
 end
+
+# Add i18n tasks (gettext:pot, gettext:po*)
```

Comment:
This calls out to the gettext gem in order to add a couple of tasks. They have to be invoked manually to generate the pot file